tutorbackup/templates/backup/build/backup/backup_services.py

```diff
@@ -45,9 +45,16 @@ def mysqldump():
     port = ENV['MYSQL_PORT']
     outfile = MYSQL_DUMPFILE
 
+    mysql_databases = ENV.get('MYSQL_DATABASES')
+    if mysql_databases:
+        databases_statement = f"--databases {mysql_databases}"
```

Comment:
`mysqldump --databases 'db_1 db_2'` breaks. You need to remove the single quotes from the `mysql_databases` string. It should result in `mysqldump --databases db_1 db_2`

Reply:
done